test/factory.js

```diff
@@ -11,6 +11,27 @@ const Person = require('../api/models/person');
 const Organisation = require('../api/models/organisation');
 const Source = require('../api/models/source');
 const Record = require('../api/models/record');
+const Publication = require('../api/models/publication');
+
+factory.define('publication', Publication, {
+  id: () => uuid.v1(),
+  created_at: new Date('2016-01-01'),
+  updated_at: new Date('2016-01-03'),
```

Comment:
Don't define the `created_at` and `updated_at` in factories, as these are overwritten by `Bookshelf` anyway...